scripts/update_feconf.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper script used for updating feconf.
+
+ONLY RELEASE COORDINATORS SHOULD USE THIS SCRIPT.
+
+Usage: Run this script from your oppia root folder:
+
+    python scripts/update_feconf.py
+"""
+
+import os
+
+import common # pylint: disable=relative-import
+
+
+FECONF_CONFIG_PATH = os.path.join(
+    os.getcwd(), os.pardir, 'release-scripts', 'feconf_updates.config')
+LOCAL_FECONF_PATH = os.path.join(os.getcwd(), 'feconf.py')
+
+
+def _apply_changes_to_feconf():
+    """Each line of the file should have a single '=' character and match a
+    line in feconf.py.
+    """
+    with open(FECONF_CONFIG_PATH, 'r') as f1:
+        config_lines = f1.read().strip().split('\n')
+
+    with open(LOCAL_FECONF_PATH, 'r') as f2:
+        feconf_lines = f2.read().strip().split('\n')
+
+    # First, verify the config file.
+    feconf_line_numbers = []
+    for config_line in config_lines:
+        assert config_line.count('=') == 1, (
+            'Invalid line in feconf config file: %s' % config_line)
+        config_line_parts = config_line.split('=')
+        matching_feconf_line_numbers = [
+            feconf_line_number
+            for (feconf_line_number, feconf_line) in enumerate(feconf_lines)
+            if feconf_line.startswith(config_line_parts[0])]
```

Comment:
Consider trimming the ``config_line_parts[0]`` or splitting by '\s*=\s*' since I think you have spaces included currently.

Reply:
This is also intentional; I intend to keep the space in order to avoid overlap with prefixes.

Reply:
Ah, can you add a comment to that affect then? It seems like it might be an error as the code currently stands.

Reply:
Done, and added an assert as well.